### PR TITLE
feat:implemented prime function

### PR DESCRIPTION
**Pull request by albin1616**
implemented prime function

### CODE REVIEW

File: training.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+function isPrime(number) {
```
Comment (Rajharsh2) [inline training.js:1]: Please add some Jsdoc.

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

File: training.js (hunk 1 above)
Comment (albin1616) [reply training.js:6]: resolved

File: training.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ */
+function isPrime(number) {
+  if (number <= 1) {
+    return false;
+  }
+
+  if (number <= 3) {
+    return true;
+  }
+
```
Comment (Rajharsh2) [inline training.js:14]: put comments where necessary.

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

File: training.js (hunk 2 above)
Comment (albin1616) [reply training.js:14]: resolved

File: training.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * 
+ * this function is used for check whwther it is prime or not
```
Comment (manvithalva12) [inline training.js:2]: please resolve the typos

**Review by manvithalva12 — CHANGES_REQUESTED** [review]

**Review by Rajharsh2 — APPROVED** [review]
LGTM

**Review by manvithalva12 — APPROVED** [review]

**Pull request merged**